src/local/butler/scripts/setup.py

```diff
@@ -37,6 +37,20 @@
 THREAD_DELAY = 30.0
 """
 
+SYZKALLER_TEMPLATE = """MAX_FUZZ_THREADS = 1
+MAX_TESTCASES = 4
+FUZZ_TEST_TIMEOUT = 4800
+TEST_TIMEOUT = 65
+WARMUP_TIMEOUT = 65
+BAD_BUILD_CHECK = False
+THREAD_ALIVE_CHECK_INTERVAL = 1
+REPORT_OOMS_AND_HANGS = True
+CORPUS_FUZZER_NAME_OVERRIDE = syzkaller
```

Comment:
Skip CORPUS_FUZZER_NAME_OVERRIDE as it is only needed when you need to change it.

Reply:
Done.